network_analysis/3B_community_detection_louvain.py

```diff
@@ -0,0 +1,202 @@
+# ---
```

Comment:
Do you want to generate a Python script for `network_analysis/3A_community_detection_planted_partition.ipynb` too? Or did you not change it because we're only using the Louvain results in the main paper?

Reply:
Since the results were similar for the two different methods, I just chose to show the louvain one in the paper. Thanks for checking